### PR TITLE
Ops-Billing UI improvements

### PR DESCRIPTION
**Pull request by nickpell**
screen shot 2016-05-03 at 1 06 36 pm

@biyeun cc: @esoergel

### CODE REVIEW

File: corehq/apps/accounting/interface.py [hunk 1]
```diff
@@ -1157,10 +1168,7 @@ def _credit_adjustment_to_row(credit_adj):
                     date_created__lte=credit_adj.date_created,
                 ))),
                 credit_adj.web_user,
-            ] + (
-                _get_credit_line_columns_from_credit_line(credit_adj.related_credit)
-                if credit_adj.related_credit else ['', '', '']
-            ))
+            ] + _get_credit_line_columns_from_credit_line(credit_adj.related_credit))
```
Comment (esoergel) [inline corehq/apps/accounting/interface.py]: :+1:

**Comment by esoergel** [comment]
The code looks good.  I don't understand the feature well enough to comment on the UI.

**Comment by nickpell** [comment]
bump @biyeun

**Comment by biyeun** [comment]
looks good!

**Pull request merged**